frontend/src/scenes/dashboard/dashboardLogic.tsx

```diff
@@ -1034,7 +1049,7 @@ export const dashboardLogic = kea<dashboardLogicType>([
                             'force_cache',
                             methodOptions
                         )
-                        dashboardsModel.actions.updateDashboardInsight(polledInsight)
+                        dashboardsModel.actions.updateDashboardInsight(polledInsight!)
```

Comment:
not following why we suddenly need the non null assertion here 😅

Reply:
I changed the type of the `getSingleInsight` function (see line 134) above to be more in line with what it actually returns. The insight is only null if something went wrong and I didn't add error handling (we didn't have it before as well).

Reply:
aah missed that, thought it was unchanged. Thanks for explaining!